docs/awscli.md

```diff
@@ -1,30 +1,13 @@
 # Install AWS CLI
 
 ```shell
-$ curl -sL 'https://github.com/Termux-pod/termux-pod/raw/main/aarch64/python/python-3.9.7/python_3.9.7_aarch64.deb' -o python_3.9.7_aarch64.deb
-$ dpkg -i python_3.9.7_aarch64.deb
-$ git clone https://github.com/aws/aws-cli.git
-$ cd aws-cli
-$ git checkout v2
+$ pkg install git python rust build-essential
+$ git clone -b v2 https://github.com/aws/aws-cli.git
 $ python -m venv venv
```

Comment:
This _mostly_ looks good to me, I like these simplifications. What is the virtual environment needed for? Aren't these instructions indicating how to build and install aws-cli v2 in a termux environment? It looks to me like you're installing into a virtual environment.

Reply:
@timblaktu I prefer using a virtual environment to avoid polluting my system.
Also, this helps me avoid running into dependency hell in case 2 different projects have similar dependencies but require incompatible versions between them.

Reply:
Thats sensible - I'm using poetry for python project deps.